### DISCUSSION TITLE
About Structured Data

### DISCUSSION THREAD

**Discussion by NicolasMassart** [squidfunk/mkdocs-material#2267] [Q&A] [answered]
I will start working on extending the templates to allot JSON-LD structured data on our sites.
I wanted to make sure it's still [out of the planned scope of your work](https://github.com/squidfunk/mkdocs-material/issues/663) before I begin.
@squidfunk can you confirm please?
Thanks.

**Comment by squidfunk** [comment, marked as answer]
Yes, as it's too custom to be made configurable.
